src/Config/ObjectModel/RuntimeConfig.cs

```diff
@@ -130,7 +130,7 @@ public bool AllowIntrospection
         }
     }
 
-    private string _defaultDataSourceName;
+    public string DefaultDataSourceName;
```

Comment:
This seems to be only public member not following getter setter syntax. `{ get; set; }` any reason not to?